install_config/registry/accessing_registry.adoc

```diff
@@ -187,17 +187,10 @@ To log in to the registry directly:
 $ oc login
 ----
 
-. Get your access token:
+. Log in to the Docker registry by using your access token:
 +
 ----
-$ oc whoami -t
-----
-
-. Log in to the Docker registry:
-+
-----
-$ docker login -u <username> -e <any_email_address> \
-    -p <token_value> <registry_ip>:<port>
+docker login -u $(oc whoami) -e <any_email_address> -p $(oc whoami -t) <registry_ip>:<port>
```

Comment:
I'd ditch the `-e <any_email_address>`. It has been deprecated for some time. And it's no longer supported since docker 17.